db/migrate/20210812033033_add_multipart_and_size_columns_to_external_upload_stubs.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class AddMultipartAndSizeColumnsToExternalUploadStubs < ActiveRecord::Migration[6.1]
+  def up
+    add_column :external_upload_stubs, :multipart, :boolean, default: false, null: false
+    add_column :external_upload_stubs, :external_upload_identifier, :string, null: true, index: true
```

Comment:
Should this be an `:bigint` column type instead? I see in the code that we're using it to store upload ids. Also, we specify `index: true` here and have a migration later on that adds on index to this column.

Reply:
No, this stores the S3 multipart "upload ID" which is a big ol' random string. I am avoiding using S3-specific terms where possible. I will delete the extra index migration, I didn't realise I doubled that up :+1: